src/p2p/net_node.h

```diff
@@ -111,15 +111,11 @@ namespace nodetool
   struct p2p_connection_context_t: base_type //t_payload_net_handler::connection_context //public net_utils::connection_context_base
   {
     p2p_connection_context_t()
-      : fluff_txs(),
-        flush_time(std::chrono::steady_clock::time_point::max()),
-        peer_id(0),
+      : peer_id(0),
```

Comment:
`sent_addresses` only works because it is only modified in receive functions that are in a strand, and the single `operator=` call also happens to only occur in that same strand. Add a comment about this, as its far easier to botch than the Dandelion++ fields that were removed.